### PR TITLE
Cool aws services

### PR DESCRIPTION
**Pull request by uolter**
- [x] I've added a short description of the entry that
      clarifies what it does and why it's useful
- [x] I've added links to the entry's documentation and/or
      website
- [x] My frontmatter is correct, complete and follows the
      format specified in the [README](../README.md)
- [x] In case the entry is in the `adopt` category: I've
      added at least one reference to showcase the entry usage
      among our organization
- [ ] I've described at least one use case for the entry

### CODE REVIEW

File: radar/platforms-and-services/aws-eventbridge.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: "AWS Lambda"
```
Comment (gunzip) [inline radar/platforms-and-services/aws-eventbridge.md]: AWS EventBridge?

**Review by gunzip — COMMENTED** [review]
We need to put each item in its own PR otherwise by no chance to discuss (and eventually approve) each one separately.

**Comment by uolter** [comment]
> We need to put each item in its own PR otherwise by no chance to discuss (and eventually approve) each one separately.

No please. I'm the lazy one :-) There're just 5 files Amazon Q wrote for me.

**Comment by gunzip** [comment]
In this case we should find someone between the codeowners in the cloud&platform area that approves all of them as a whole.

**Comment by uolter** [comment]
> In this case we should find someone between the codeowners in the cloud&platform area that approves all of them as a whole.

@Krusty93  please :-) Let me buy you a beer.

**Review by Krusty93 — APPROVED** [review]

**Pull request merged**